linkerd/system/src/linux.rs

```diff
@@ -24,8 +27,8 @@ pub fn ms_per_tick() -> io::Result<u64> {
     Ok(ms_per_tick)
 }
 
-pub fn blocking_stat() -> io::Result<Stat> {
-    pid::stat_self()
+pub fn blocking_stat() -> ProcResult<Stat> {
+    Process::myself().unwrap().stat()
```

Comment:
We shouldn't probably just unwrap here especially since it anyways returns a `Result`.

```suggestion
    Process::myself()?.stat()
```